.github/workflows/docker.yml

```diff
@@ -0,0 +1,84 @@
+name: Create and publish a Docker images
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+    branches:
+      - "master"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    name: PHP ${{ matrix.php-version }}, Fixer ${{ github.ref == 'refs/heads/master' && 'edge' || github.ref_name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.4'
+            php-suffix: '-php7.4'
+            alpine-version: '3.16'
+          - php-version: '8.0'
+            php-suffix: '-php8.0'
+            alpine-version: '3.16'
+          - php-version: '8.1'
+            php-suffix: '-php8.1'
+            alpine-version: '3.18'
+          - php-version: '8.2'
+            php-suffix: '-php8.2'
+            alpine-version: '3.18'
+          - php-version: '8.3'
+            php-suffix: '-php8.3'
+            alpine-version: '3.18'
+          # This is the latest supported PHP Version and will be provided without -phpX.X suffix
+          - php-version: '8.3'
+            php-suffix: ''
+            alpine-version: '3.18'
```

Comment:
This has 2 drawbacks IMHO, connected to each other - it runs two jobs for 8.3 which is displayed poorly at the moment:

<img width="332" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/aa4e33d0-c7d8-4170-adac-011f26b40528">

I believe that with [this change](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7555#pullrequestreview-1790631686) we can get rid of this part and `php-suffix` altogether.

Reply:
It'll be fixed after resolving [this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7555#discussion_r1432843243), so I'm going to close this discussion and keep only that one.